### ISSUE TITLE
Unable to upvote post when on comments page

### ISSUE THREAD

**Issue by davidbernstein88** [christianselig/apollo-bugs#906]
**Describe the bug**
I am able to upvote a post when scrolling by posts, but once I select a post, I cannot upvote it from the comments page. See video for behavior.

**To Reproduce**
1. Open app
2. Touch a post
3. Touch the upvote button

**iOS version**: iOS 13.5.1
**Apollo version**: 1.8.5, PRO Enabled
**iOS device**: iPhone 11 Pro
**Jailbroken**: No
**Reproducible:** 100%

**Expected behavior**
Upvoting should be enabled

**Video**
![upvote_bug](https://user-images.githubusercontent.com/9099322/87994595-5dd29b80-cabb-11ea-97d3-4a8773f26546.gif)

**Comment by ahhhhu** [comment]
Yeah I don't have that issue I can upvote without problem I have iOS 14 beta
